dev-lang/janet/Manifest

```diff
@@ -0,0 +1 @@
+DIST janet-1.6.0.tar.gz 956098 BLAKE2B 3555419ec24cedbd093f0b156702dbdf58cc9f0efcf52b87f49365eea69b1e10193d82b09d2574df4b5026e3f6e33edf14fc20eecc59f0106220f2b81c916bcb SHA512 9b4707a5121fdee887810b0f652c1acb4059ee6f802d9118c3fdc20421f660f0071d7833a2a6a5c929878d21216a34ee6af2ea190061c94b7b8585eca3d0f57a
```

Comment:
1.7.0 is available. Upstream also seems to have meson build option available, could you switch to use that instead?

Reply:
It's not officially supported, and I  am not familiar with meson ebuilds...